modules/appeals_api/spec/sidekiq/appeal_submitted_job_spec.rb

```diff
@@ -0,0 +1,228 @@
+      sc_template_name.to_sym => sc_template_id,
+      :"#{hlr_template_name}_claimant" => claimant_hlr_template_id,
+      :"#{nod_template_name}_claimant" => claimant_nod_template_id,
+      :"#{sc_template_name}_claimant" => claimant_sc_template_id
+    }]
+  end
+
+  describe 'perform' do
+    let(:vanotify_client) { instance_double(VaNotify::Service) }
+    let(:appeal_id) { appeal.id }
+    let(:appeal_class_str) { appeal.class.name }
+    let(:date_submitted_str) { DateTime.new(2024, 1, 2, 3, 4, 5).iso8601 }
+
+    describe 'successes' do
+      let(:expected_tags) { { appeal_type: 'hlr', claimant_type: 'veteran' } }
+      let(:expected_date_submitted) { 'January 02, 2024' }
+
+      before do
+        allow(FeatureFlipper).to receive(:send_email?).and_return(true)
+        allow(VaNotify::Service).to receive(:new).and_return(vanotify_client)
+        allow(vanotify_client).to receive(:send_email)
+        allow(StatsD).to receive(:increment).with(described_class::STATSD_CLAIMANT_EMAIL_SENT, tags: expected_tags)
+        with_settings(*settings_args) { job.perform(appeal_id, appeal_class_str, date_submitted_str) }
+      end
+
+      context 'with HLR' do
+        it 'sends HLR email to veteran' do
+          expect(vanotify_client).to have_received(:send_email).with(
+            {
+              email_address: appeal.veteran.email,
+              personalisation: {
+                date_submitted: expected_date_submitted,
+                first_name: appeal.veteran.first_name
+              },
+              template_id: hlr_template_id
+            }
+          )
+        end
+
+        context 'with non-veteran claimant' do
+          let(:appeal) { create(:extra_higher_level_review_v2) }
+          let(:expected_tags) { { appeal_type: 'hlr', claimant_type: 'non-veteran' } }
+
+          it 'sends HLR email to non-veteran claimant' do
+            expect(vanotify_client).to have_received(:send_email).with(
+              {
+                email_address: appeal.claimant.email,
+                personalisation: {
+                  date_submitted: expected_date_submitted,
+                  first_name: appeal.claimant.first_name,
+                  veterans_name: appeal.veteran.first_name
+                },
+                template_id: claimant_hlr_template_id
+              }
+            )
+          end
+        end
+
+        context 'if for some hypothetical reason there is no email' do
+          let(:form_data) do
+            data = fixture_as_json('decision_reviews/v2/valid_200996.json')
+            data['data']['attributes']['veteran'].delete('email')
+            data
+          end
+          let(:appeal) { create(:higher_level_review_v2, form_data:) }
+
+          it 'invokes VANotify with ICN instead' do
+            expect(vanotify_client).to have_received(:send_email).with(
+              {
+                recipient_identifier: { id_type: 'ICN', id_value: appeal.veteran_icn },
```

Comment:
I was curious about `recipient_identifier` here, because it's not in [the original docs for the notify API project](https://docs.notifications.service.gov.uk/ruby.html). I learned that this is actually something VA added in its fork of that project. You can see it in [the OpenAPI spec for the VANotify API](https://github.com/department-of-veterans-affairs/notification-api/blob/master/documents/openapi/openapi.yaml), which you can view in a browser by pasting the yaml into a viewer like https://editor.swagger.io.